src/UI/Art/SpriteArtist/SpriteMapView.java

```diff
@@ -119,18 +119,6 @@ public void setPreferredDimensions(int width, int height)
     mapTilesToDrawX = mapViewWidth / tileSize;
     mapTilesToDrawY = mapViewHeight / tileSize;
 
-    // Cap the view width/height to the dimensions of the map.
-    if( mapTilesToDrawX > myGame.gameMap.mapWidth )
-    {
-      mapTilesToDrawX = myGame.gameMap.mapWidth;
-      mapViewWidth = mapTilesToDrawX * tileSize;
-    }
-    if( mapTilesToDrawY > myGame.gameMap.mapHeight )
-    {
-      mapTilesToDrawY = myGame.gameMap.mapHeight;
-      mapViewHeight = mapTilesToDrawY * tileSize;
```

Comment:
I get that folks want to be able to make it full-screen, but I also kind of want to preserve a way to set the window to the map size. Didn't you make a change somewhere that the window would size to the map when starting the game, and just allow resizing after that?

Reply:
Totally did. Beats me what PR it's in, though.

Reply:
....and *that'd* be why. It's this PR. https://github.com/ThislsAUsername/DefendPeace/pull/192/commits/216db00a2eb0f7df9ce6e9ef219cc685eb793f81